templates/_linux.yaml

```diff
@@ -66,7 +70,16 @@ objects:
             - disk:
                 bus: {{ diskbus | default("virtio") }}
               name: rootdisk
+            - disk:
+                bus: {{ diskbus | default("virtio") }}
+              name: cloudinitdisk
+            interfaces:
+            - bridge: {}
+              name: default
         terminationGracePeriodSeconds: 0
+        networks:
```

Comment:
I am not sure this should be added by default. There are cases where a VM without networking might be needed.

Reply:
All of the base cloud images I've ever seen expect the VM to have one nic. The RHEL and CentOS 7 images as example have a dhcp ifcfg for eth0